arbitrator/wasm-libraries/arbkeccak/Cargo.toml

```diff
@@ -0,0 +1,11 @@
+[package]
+name = "arbkeccak"
+version = "0.1.0"
+edition = "2021"
+publish = false
+
+[lib]
+crate-type = ["cdylib"]
```

Comment:
I assume this is the only crate that pulls `cdylib`? Asking if it's worth adding to the workspace deps?